host/hackrf-tools/src/hackrf_transfer.c

```diff
@@ -749,56 +754,72 @@ int main(int argc, char** argv) {
 			usage();
 			return EXIT_FAILURE;
 		}
-		
-		if( result != HACKRF_SUCCESS ) {
-			fprintf(stderr, "argument error: '-%c %s' %s (%d)\n", opt, optarg, hackrf_error_name(result), result);
+
+		if (result != HACKRF_SUCCESS) {
+			fprintf(stderr,
+				"argument error: '-%c %s' %s (%d)\n",
+				opt,
+				optarg,
+				hackrf_error_name(result),
+				result);
 			usage();
 			return EXIT_FAILURE;
-		}		
+		}
 	}
 
-	if (lna_gain % 8)
+	if (lna_gain % 8) {
 		fprintf(stderr, "warning: lna_gain (-l) must be a multiple of 8\n");
+	}
 
-	if (vga_gain % 2)
+	if (vga_gain % 2) {
 		fprintf(stderr, "warning: vga_gain (-g) must be a multiple of 2\n");
+	}
 
 	if (samples_to_xfer >= SAMPLES_TO_XFER_MAX) {
-		fprintf(stderr, "argument error: num_samples must be less than %s/%sMio\n",
-			u64toa(SAMPLES_TO_XFER_MAX,&ascii_u64_data1),
-			u64toa((SAMPLES_TO_XFER_MAX/FREQ_ONE_MHZ),&ascii_u64_data2));
+		fprintf(stderr,
+			"argument error: num_samples must be less than %s/%sMio\n",
+			u64toa(SAMPLES_TO_XFER_MAX, &ascii_u64_data1),
+			u64toa((SAMPLES_TO_XFER_MAX / FREQ_ONE_MHZ), &ascii_u64_data2));
 		usage();
 		return EXIT_FAILURE;
 	}
 
 	if (if_freq || lo_freq || image_reject) {
 		/* explicit tuning selected */
 		if (!if_freq) {
-			fprintf(stderr, "argument error: if_freq_hz must be specified for explicit tuning.\n");
+			fprintf(stderr,
+				"argument error: if_freq_hz must be specified for explicit tuning.\n"
+			);
 			usage();
 			return EXIT_FAILURE;
 		}
 		if (!image_reject) {
-			fprintf(stderr, "argument error: image_reject must be specified for explicit tuning.\n");
+			fprintf(stderr,
```

Comment:
Might there be a setting to not reformat fprintf's at all?
Maybe until someone comes up with a better algorithm. ;)

Reply:
No, clang-format doesn't give that level of control. But see note above about using a macro/wrapper for these.